bee-message/tests/ed25519_address.rs

```diff
@@ -40,7 +40,7 @@ fn to_string() {
     bytes.copy_from_slice(&vec);
     let address = Ed25519Address::new(bytes);
     assert_eq!(
-        address.to_string(),
+        address.to_bech32(),
```

Comment:
That's now how it should be fixed. This is a test specific for to_string so you need to change the string we compare to.

Reply:
oops